src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -817,6 +816,26 @@ def collect_periscope_logs(resource_group_name, name, storage_account_name=None,
     else:
         display_diagnostics_report(kubectl_prior)
 
+    try:
+        print("Deleting aks-periscope resources from cluster ...")
+
+        subprocess_cmd = kubectl_prior + ["delete", "serviceaccount,configmap,daemonset,secret", "--all", "-n", "aks-periscope", "--ignore-not-found"]
+        subprocess.call(subprocess_cmd, stderr=subprocess.STDOUT)
+
+        subprocess_cmd = kubectl_prior + ["delete", "ClusterRoleBinding", "aks-periscope-role-binding", "--ignore-not-found"]
+        subprocess.call(subprocess_cmd, stderr=subprocess.STDOUT)
+
+        subprocess_cmd = kubectl_prior + ["delete", "ClusterRole", "aks-periscope-role", "--ignore-not-found"]
+        subprocess.call(subprocess_cmd, stderr=subprocess.STDOUT)
+
+        subprocess_cmd = kubectl_prior + ["delete", "--all", "apd", "-n", "aks-periscope", "--ignore-not-found"]
+        subprocess.call(subprocess_cmd, stderr=subprocess.STDOUT)
+
+        subprocess_cmd = kubectl_prior + ["delete", "CustomResourceDefinition", "diagnostics.aks-periscope.azure.github.com", "--ignore-not-found"]
+        subprocess.call(subprocess_cmd, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError as err:
```

Comment:
lets's add a broader generic Exception below, just for catching any unexpected exception.